build.py

```diff
@@ -0,0 +1 @@
+exit(0)
```

Comment:
This is really just a hack to make poetry spit out a wheel with build tags. Would it make sense to at least spit out some info here saying "lolz - not actually compiling anything - you'll need a global install"?